tutorials/developer-tutorials/add-environment.md

```diff
@@ -65,8 +67,19 @@ You will now have a new wallet called `ibp-wallet` containing the `Org1 Admin` i
 4. You will then be asked for the method for adding an identity chose `Provide JSON identity file from IBM Blockchain Plaform`.
 
 6. Browse to the location of the admin file you exported previously and select the JSON file.
- 
-You may be wondering why we don't have an identity file for the certificate authority. This is because the JSON for the certificate authority contains `enroll id` and `enroll secret` properties. These can be used to connect to the certificate authority node.
+
+Finally you now need to add the identity for the certificate authority
+
+1. On the `Wallets` panel right click on `ibp-wallet` and choose `Add identity to wallet`.
+
+2. You will the be asked to provide a name for the identity use `Org1 CA Admin`.
+
+3. You will then be asked to provide an MSPID use `org1msp`.
+
+4. You will then be asked for the method for adding an identity chose `Provide JSON identity file from IBM Blockchain Plaform`.
```

Comment:
"4. When asked for the method of adding an identity, select `Provide JSON identity file from IBM Blockchain Plaform`."